src/Microsoft.ApplicationInsights.AspNetCore/Extensions/ApplicationInsightsServiceOptions.cs

```diff
@@ -17,6 +17,7 @@ public ApplicationInsightsServiceOptions()
             this.EnableQuickPulseMetricStream = true;
             this.EnableAdaptiveSampling = true;
             this.EnableDebugLogger = true;
+            this.EnableAuthenticationTracking = false;
```

Comment:
Don't think we have a unit test to validate the defaults are as we intent it to be.
Would be great if you can add one, but its okay to skip it as I can create a separate PR to address this.

Reply:
Yeah, I was also wondering how to test defaults. It seems there is no test for it.

I'd prefer if we could address this issue at a later time so we don't artificially prolong the discussions about this PR. We could either directly do it in a separate one but I also wouldn't mind to do it in this PR - when all other issues are resolved. 😊